image/src/routes/metadata.ts

```diff
@@ -59,19 +59,19 @@ export const getMetadata = async (c: HonoInterface) => {
     const normalized = normalize(content, ipfsUrl)
 
     const image = normalized.image
-    const imageMetadata = await getMimeType(image)
+    const imageMimeType = await getMimeType(image)
 
     const animationUrl = normalized.animationUrl
-    let animationUrlMetadata
+    let animationUrlMimeType
     if (animationUrl) {
-      animationUrlMetadata = await getMimeType(animationUrl)
+      animationUrlMimeType = await getMimeType(animationUrl)
     }
 
     const predefinedAttributes = {
       image,
-      imageMetadata,
+      imageMimeType,
       animationUrl,
-      animationUrlMetadata,
+      animationUrlMimeType,
```

Comment:
```suggestion
      mimeType: animationUrlMimeType,
```

Reply:
we need both mimeType actually, for image and animationUrl